frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.tsx

```diff
@@ -67,16 +85,35 @@ export const ProjectsList: FC<IProjectsListProps> = ({ projects, project }) => {
         return <LinkCell to={`/projects/${item}`} title={item} />;
     }
 
+    const tokenFormat = secret?.includes(':') ? 'v2' : 'v1'; // see https://docs.getunleash.io/reference/api-tokens-and-client-keys#format
+    const isWildcardToken = secret?.startsWith('*:');
+    const isOrphanedToken = tokenFormat === 'v2' && !isWildcardToken;
+
     return (
         <TextCell>
             <HtmlTooltip
-                title='ALL current and future projects'
+                title={
+                    isOrphanedToken ? (
+                        <>
+                            ALL current and future projects. This is an orphaned
+                            token with it's project deleted. This token gives
+                            more access than it was intended to and should be
+                            deleted.
```

Comment:
I wonder if we can rephrase this somehow. Because from the way it sounds now, it seems like it's a bug in Unleash (which it is), but it also sounds like we're not going to fix it.

I wonder if we can phrase this as a warning of sorts - like "it works now, but it'll stop working in the future"?

I don't know. Something like:

```suggestion
                            This is an orphaned token. All of its original projects have been deleted and it now has access to all current and future projects. You should stop using this token and delete it. It will lose access to all projects at a later date.
```